src/event-helper.js

```diff
@@ -170,8 +173,14 @@ export function loadPromise(eleOrWindow) {
   if (isLoaded(eleOrWindow)) {
     return Promise.resolve(eleOrWindow);
   }
+  const isMediaElement = isHTMLMediaElement(eleOrWindow);
+  if (
+    isMediaElement &&
+    eleOrWindow[MEDIA_LOAD_FAILURE_SRC_PROPERTY] === eleOrWindow.currentSrc
```

Comment:
Can't the source be changed between when the error happens and when this executes?  Wouldn't that lead us to drop errors?

Reply:
If the source changes that means the load is retried, so we want that test to fail to start listening for load/error again. I believe this is WAI, and copying the `HTMLMediaElement.error` behavior.